src/main/java/com/epam/rest/HelloWorldService.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.rest;
+
+import com.epam.dao.UserDao;
+import com.epam.entities.User;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/users")
+public class HelloWorldService {
+    private UserDao userDao = new UserDao();
+
+    @POST
+    @Path("/create")
```

Comment:
You don't need to mention that it's *create* or *delete* you have REST methods for that - @POST, @DELETE, @PUT, @GET